Aviao.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Classe Aviao
+ */
+public class Aviao implements VeiculoInterface {
+
+	private double preco;
+	private String marca;
+	private int ano;
+	private String modelo;
+	private String consumo;
+	private int alturaMaxima;
+	private int capacidadePassageiros;
+	private int capacidadeCarga;
+	private String tipo;
+
+	// Construtor
+	public Aviao(Double preco, String marca, int ano, String modelo, String consumo, int alturaMaxima, int capacidadePassageiros, int capacidadeCarga, String tipo){
+		this.preco = preco;
+		this.marca = marca;
+		this.ano = ano;
+		this.modelo = modelo;
+		this.consumo = consumo;
+		this.alturaMaxima = alturaMaxima;
+		this.capacidadePassageiros = capacidadePassageiros;
+		this.capacidadeCarga = capacidadeCarga;
+		this.tipo = tipo;
+	}
+
+	// Getters e setters
+	public Double getPreco(){
+		return this.preco;
+	}
+	public void setPreco(Double novoPreco){
+		this.preco = novoPreco;
+	}
+
+	public String getMarca(){
+		return this.marca;
+	}
+	public void setMarca(String novaMarca){
+		this.marca = novaMarca;
+	}
+
+	public int getAno(){
+		return this.ano;
+	}
+	public void setAno(int novoAno){
+		this.ano = novoAno;
+	}
+
+	public String getModelo(){
+		return this.modelo;
+	}
+	public void setModelo(String novoModelo){
+		this.modelo = novoModelo;
+	}
+
+	public String getConsumo(){
+		return this.consumo;
+	}
+	public void setConsumo(String kmPorLitro){
+		this.consumo = kmPorLitro;
```

Comment:
A interface estava com o tipo errado, já foi corrigido então só dar um merge da tua branch com o master e troca o tipo para Double.

```suggestion
	public void setConsumo(Double kmPorLitro){
		this.consumo = kmPorLitro;
```